Assets/Scripts/EOSLobbyManager.cs

```diff
@@ -48,6 +48,12 @@ public class Lobby
         public bool AllowInvites = true;
         public bool? DisableHostMigration;
 
+        /// <summary>
+        /// Has the local user deafened themself?
+        /// This doesn't carry over between rooms.
```

Comment:
nit: Change to "Indicates whether the local user has deafened themselves (this doesn't carry over between rooms)."

Reply:
Implemented as suggested. (plus line break)